www/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow: /draft/
+Disallow: /examples/
```

Comment:
Why disallow /examples?

Reply:
Pretty funny story, but Google thinks some of them are "Soft 404s" because they contain just the word "Test" or something similar. I guess we could alternatively add more text to them to convince Google otherwise, but that seems counter to the actual purpose of the example.